Essentials/src/com/earth2me/essentials/commands/Commandjump.java

```diff
@@ -46,7 +47,12 @@ public void run(final Server server, final User user, final String commandLabel,
 
         final Trade charge = new Trade(this.getName(), ess);
         charge.isAffordableFor(user);
-        user.getTeleport().teleport(loc, charge, TeleportCause.COMMAND);
+        // If the user is allowed to jump outside of the vanilla world boundary
+        if(Bukkit.getWorld(user.getWorld().getUID()).getWorldBorder().isInside(loc) || user.isAuthorized("essentials.jump.pastWorldBorder")) {
+            user.getTeleport().teleport(loc, charge, TeleportCause.COMMAND);
+        }else {
+            user.sendMessage("You can not jump outside of the world border");
```

Comment:
Messages in Essentials should be configurable. You can look at some other code or pull requests in Essentials to get an idea of how you should accomplish this. It is not too hard, and many servers will want to change the message to something else.